client/src/Component/MainTable.js

```diff
@@ -75,14 +74,48 @@ const MainTable = (props) => {
 					{clientData.map((client) => {
 						return (
 							<tr key={client.client_id}>
-								<td><Checkbox isChecked={props.checkedCheckboxes.includes(client.client_id)} label={client.name} onChange={(checked) => props.onHandleSelectedUserState(checked, client.client_id)} /></td>
+								<td>
+									<Checkbox
+										isChecked={props.checkedCheckboxes.includes(
+											client.client_id
+										)}
+										label={client.name}
+										onChange={(checked) =>
+											props.onHandleSelectedUserState(checked, client.client_id)
+										}
+									/>
+								</td>
 								<td>{client.client_id}</td>
 								<td>{new Date(client.date_added).toLocaleString()}</td>
 								<td>{client.name}</td>
 								<td>{client.phone_number}</td>
 								<td>{client.bikes_needed}</td>
-								<td>{client.booking_status}</td>
-								<td>{client.residency_status}</td>
+								<td>
+									{client.booking_status == 1
```

Comment:
We usually try not to have big JS expressions in the React template—can we move this logic to a function instead? Or instead, we could create an object like 
```js
const bookingStatus = {
  1: 'Waiting',
  2: 'Contacted',
  ...
}
```
And then do something like `bookingStatus[client.booking_status]` to convert the status to the text.